public/proxy_pages/invalid_sso_credentials.js

```diff
@@ -0,0 +1,6 @@
+$('#invalid_sso_credentials_flash').click(function() {
+  $(this).text('');
```

Comment:
`$(this).hide()` instead? Otherwise, if the login fails again, they'll see an empty error? Unless it's always repainted..